### PR TITLE
Create multi-stage build.

### PR DESCRIPTION
**Pull request by verdammelt**
First part builds a 'representer' binary which is then used in the
runtime image.

The resulting image size is slightly smaller and the runtime is a
faster too.

### CODE REVIEW

**Comment by verdammelt** [comment]
NOTE This PR is built on top of PR #11 & PR #10.

**Review by TheLostLambda — APPROVED** [review]
I think this is a really good structure for things! I was going for as simple as possible with the test-runner, but I think it's time we move to something a bit more "best-practices". I think this (and the parent PRs) look great and if you'd like to do something similar for the test-runner, feel free!

File: Dockerfile [hunk 1]
```diff
@@ -17,13 +18,16 @@ COPY build/build.lisp build/
 COPY src quicklisp/local-projects/representer
 RUN sbcl --script ./build/build.lisp
 
+## Build the runtime image
+FROM alpine
+WORKDIR /opt/representer
+
 # Copy over the test runner
```
[the remark this answers is not in the document]
Comment (verdammelt) [reply Dockerfile:25]: haha - been looking at this file for a while and never noticed that!

I guess it is true that comments are always wrong :)

[the remark this answers is not in the document]
Comment (verdammelt) [reply Dockerfile:25]: > "best-practices"

Yeah, what we have is "perfectly fine (probably) practices" so let's upgrade to "pretty good practices" at least :)

**Pull request merged**